### ISSUE TITLE
Is this inspired from https://ayushk7.github.io/CodeWire/ ?

### ISSUE THREAD

**Issue by ayushk7** [moonrailgun/codeck#4]
Hi @moonrailgun,
This looks nicely implemented better then my implementation of the [CodeWire](https://ayushk7.github.io/CodeWire/).
I just wanted to confirm if you've taken inspiration from the CodeWire, as I can see at certain places, you're using similar code and structure.

Anyway, codeck looks awesome.

**Comment by moonrailgun** [comment]
Thank you for your interest! Codeck was mainly inspired by Unreal’s Blueprint system, as I am an Unreal developer and am also working on an Unreal-based game project.

I find the design of Blueprints very useful—it’s much more appealing compared to Blockly. The main flow design is more attractive for code-based applications (as opposed to traditional design).

So, when I was searching for something similar on the web, my priority was to avoid reinventing the wheel. If a project is good enough, I’d be more than willing to join in. Consequently, I researched all related node editor projects on GitHub, including Rete, BaklavaJS, Blackprint, and, of course, CodeWire.

To my knowledge, I’ve referenced some implementations available in the community. CodeWire is a great project, and it was certainly a learning resource for me.

So, regarding whether I referenced CodeWire, my answer is that my main inspiration was Unreal’s Blueprint system. For some implementations, I did look at community resources, which certainly includes CodeWire. I can’t recall the specific aspects I referenced, as it’s been quite a while. You’ve probably noticed that I haven’t updated Codeck for over a year.

I hope this answers your question?
